### DISCUSSION TITLE
PocketBase Sync: Synchronize Individual Collection Schema & Data between PocketBase Instances

### DISCUSSION THREAD

**Discussion by Mashiane** [pockethost/pockethost#208] [Show and tell]
Hi there

I usually develop my and test my PocketBase Applications locally and when I need to present them, move the schemas to PocketHost.
With PocketHost, one is able to create different instances of databases whilst locally on a PC, you just have 1 PocketBase instance.

Currently, if you Export and Import collections, you are faced with deleting or even overwriting content. I wanted a tool that could help me sync any schema of choice between PocketHost and a locally running PocketBase instance.

This is developed using TailwindCSS and DaisyUI. Check it out.

https://candid-daffodil-9a4d11.netlify.app/

https://youtu.be/RNdVqBs7Xh0

**Comment by Mashiane** [comment]
You can now read about this on Medium

https://medium.com/javascript-in-plain-english/how-to-sync-individual-pocketbase-collection-schemas-and-data-d7db1647d730

**Comment by benallfree** [comment]
@Mashiane Cool idea!

1. Does this have anything to do with PocketHost specifically? I'm curious why you are posting in the PocketHost discussion area rather than PocketBase.
2. Is this an open source project? If so, please consider making a PR to https://github.com/benallfree/awesome-pocketbase

**Reply by Mashiane** [reply]
1. Yeah, I am developing my apps with a single instance of PocketBase however each time I present my work I push the data to PocketHost. I have created multiple instances in PocketHost for each app and in most instances want to keep my localDB in sync with my local PocketBase instance. What I do, in my localdb I prefix my tables so that I can link them to a PocketHost instance, then use the sync tool to sync to PocketHost. This also helps because  I can just sync 1 collection at a time etc.

**Reply by Mashiane** [reply]
2. Cool, I will do that. The app is deployed on Netlify and one can also run it locally. The repo is here: https://github.com/Mashiane/SithasoDaisy-PocketHost-Sync

**Reply by rohitkrishna094** [reply]
Cool Project @Mashiane , it worked on first try for me. you should add this to  https://github.com/benallfree/awesome-pocketbase.
